### PR TITLE
[TAS-1743] 🎨 Force refetch classes after claiming

### PR DESCRIPTION
**Pull request by AuroraHuang22**
Force-fetch collected classes when redirect back to the bookshelf.

task: [[bug] Books on the bookshelf have not been updated correctly](https://www.notion.so/likecoin/bug-Books-on-the-bookshelf-have-not-been-updated-correctly-6e7a79d25ccd473381674f4dd9b7c6ac?pvs=4)

### CODE REVIEW

File: src/pages/nft/claim/index.vue [hunk 1]
```diff
@@ -1098,11 +1098,15 @@ export default {
         this.primaryKey,
         1
       );
+      const query = { tab: 'collected' };
+      if (this.isAutoDeliver) {
+        query.refetch = true;
+      }
```
Comment (williamchong) [inline src/pages/nft/claim/index.vue:1104]: Can we trigger a `fetchCollectedNFTClassesByAddress` here instead of in next page?

Comment (AuroraHuang22) [reply src/pages/nft/claim/index.vue]: How about trigger `fetchCollectedNFTClassesByAddress` right after claiming?
[🎨 Refetch classes list after claiming](https://github.com/likecoin/liker-land/pull/1631/commits/e73ee6eab252d3953d8f359e061167bf5dd5a3fa)

**Review by williamchong — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

File: src/pages/nft/claim/index.vue [hunk 2]
```diff
@@ -777,6 +777,7 @@ export default {
     }
   },
   methods: {
+    ...mapActions(['fetchCollectedNFTClassesByAddress']),
```
Comment (nwingt) [inline src/pages/nft/claim/index.vue:780]: I think NFT mixin already provided this action

https://github.com/likecoin/liker-land/blob/08ef670f2c6a8bd6bcbe62256309f9d2a50c944f/src/mixins/nft.js#L763

Comment (AuroraHuang22) [reply src/pages/nft/claim/index.vue]: [🔥 Remove duplicated action](https://github.com/likecoin/liker-land/pull/1631/commits/eebffbf34ef7603e3a4a090d419883fc69d77146)

Comment (AuroraHuang22) [reply src/pages/nft/claim/index.vue]: [🔥 Remove duplicated action](https://github.com/likecoin/liker-land/pull/1631/commits/eebffbf34ef7603e3a4a090d419883fc69d77146)

**Review by nwingt — APPROVED** [review]

**Pull request merged**